CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+# Contributing to Newsboat
+
+Thank you for your interest in contributing to Newsboat! Here are a few ways
+you can help:
+
+1. [Feature Requests](#feature-requests)
+2. [Bug Reports](#bug-reports)
+3. [Patches / Pull Requests](#patches--pull-requests)
+    1. [Testing](#testing)
+    2. [Documentation](#documentation)
+    3. [Translations](#translations)
+
+## Feature Requests
+
+Feature requests should be reported in the
+[issue tracker](https://github.com/newsboat/newsboat/issues) or on
+the [mailing list](https://groups.google.com/group/newsboat). Ensure
+the feature request was not already reported by searching the issue tracker
+and/or mailing list. Feature requests are filed under the `enhancement` label
+on the [issue tracker](https://github.com/newsboat/newsboat/labels/enhancement).
+
+
+## Reporting Bugs
+
+**Do not open a GitHub issue if the bug is a security vulnerability**. Send an
+email to security@newsboat.org. Please encrypt your emails to 
+[PGP key 4ED6CD61932B9EBE](https://newsboat.org/newsboat.pgp)
+
+Bug reports should be reported in the
+[issue tracker](https://github.com/newsboat/newsboat/issues) or on
+the [mailing list](https://groups.google.com/group/newsboat). Again,
+please ensure the bug was not already reported.
+
+
+## Patches / Pull Requests
+
+Patches can be submitted by sending a 
+[GitHub Pull Request](https://github.com/newsboat/newsboat/pull/new/master) or 
+emailing a patch to the Newsboat 
+[mailing list](https://groups.google.com/group/newsboat). 
+To learn more about patches, see 
+[here](https://www.kernel.org/doc/html/latest/process/submitting-patches.html) and 
+[here](https://github.com/git/git/blob/master/Documentation/SubmittingPatches). To
+learn more about pull requests, 
+see the [GitHub docs](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
+
+If you are looking for a place to start contributing, take a look at the issues labeled 
+[good first issue](https://github.com/newsboat/newsboat/labels/good%20first%20issue)
+and [docs](https://github.com/newsboat/newsboat/labels/docs) on the issue tracker.
+[The Newsboat Hacker's Guide](https://github.com/newsboat/newsboat/blob/master/doc/hackers-guide.asciidoc)
+has information on Newsboat's architecture and design decisions, which might prove
+useful.
+
+Always write clear and concise commit messages describing your changes and make
+sure to reference any issues and/or other commits. An example of a well written
+commit message can be found [here](https://chris.beams.io/posts/git-commit/).
+
+Please follow our 
+[style guide](https://github.com/newsboat/newsboat/blob/master/doc/internal/code-style.markdown) 
+when contributing code.
+
+### Testing
+
+To make sure no regressions were introduced, all tests should be run before
+sending a pull request or patch. The following command can be run to test your
```

Comment:
It'd be *nice* if people ran tests beforehand, but it's not a requirement at all — we have continuous integration (and continuous testing), which will run `make check` with various different compilers on multiple operating systems.

Especially for people who only change the docs, relying on CI is way more practical than setting up a local environment (because our docs actually require not just Asciidoctor and AWK, but also C++, which ties into our config.sh script which searches for some of the libraries…)

I don't have an immediate suggestion on how to rephrase that, but I think this paragraph should be rewritten somehow.

Reply:
Maybe using the original would be better?
> You'll probably want to run the tests; here's how:

Or also mentioning that running tests locally is encouraged for code changes, not doc related changes. Maybe also mention CI running tests.

Reply:
> Or also mentioning that running tests locally is encouraged for code changes, not doc related changes. Maybe also mention CI running tests.

I think that'd be ideal. Failing that, I'm fine with re-using the original wording. We can always improve it later on, it's not like this PR has to nail the perfect wording on the first try :)